pkg/ruler/remotequerier.go

```diff
@@ -178,16 +155,16 @@ func (q *RemoteQuerier) Read(ctx context.Context, query *prompb.Query, sortSerie
 		return nil, errors.Wrapf(err, "unable to marshal read request")
 	}
 
-	req := httpgrpc.HTTPRequest{
+	req := http.Request{
```

Comment:
can we use `http.NewRequest` here? I think it does a bit more than what this function does right now

Reply:
same for createRequest below